.github/workflows/push-ingestor-build.yaml

```diff
@@ -40,3 +40,29 @@ jobs:
           tags: ${{ secrets.ECR_REPO }}:${{ github.head_ref || github.ref_name }}
           push: true
           provenance: false
+  push-ingestorv2-build:
+    name: build and push apiv2 ingestor to remote
+    runs-on: ubuntu-latest
+    environment: staging
+    if: github.repository == 'chanzuckerberg/cryoet-data-portal-backend'
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          mask-aws-account-id: true
+          aws-region: ${{ secrets.AWS_REGION }}
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-duration-seconds: 1200
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Docker build & Push
+        uses: docker/build-push-action@v4
+        with:
+          context: ./apiv2
+          file: ./apiv2/Dockerfile
```

Comment:
Not a blocker: It looks like, the only difference between the ingestion job and this one are these two values. What are your thoughts on having a composite action that can be used by both these jobs?

Reply:
I updated it to use a matrix build. check again?